lib/pure/cookies.nim

```diff
@@ -8,11 +8,14 @@
 #
 
 ## This module implements helper procs for parsing Cookies.
+## https://tools.ietf.org/html/rfc6265
 
 import strtabs, times
 
 proc parseCookies*(s: string): StringTableRef =
-  ## parses cookies into a string table.
+  ## parses cookies into a string table. Example input: "a=1; foo=bar"
```

Comment:
By the way, what is the difference between a cookie header set by a client and a server? Does that mean I can't parse a "Set-Cookie" header returned to me by a server?